vivo.owl

```diff
@@ -4791,6 +4791,16 @@ Definition source: https://ror.org/about/</obo:IAO_0000112>
     </owl:DatatypeProperty>
 
 
+    <!-- http://vivoweb.org/ontology/core#wikidataiD -->
+
+    <owl:DatatypeProperty rdf:about="http://vivoweb.org/ontology/core#wikidataiD">
+        <rdfs:label xml:lang="en">Wikidata ID</rdfs:label>
+        <obo:IAO_0000112 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Identifier being used by Wikidata to identify entities in Wikidata.org. Entity ID URIs follow the pattern http://www.wikidata.org/entity/ID where ID is an entity ID.</obo:IAO_0000112>
+        <rdfs:subPropertyOf rdf:resource="http://vivoweb.org/ontology/core#identifier"/>
+        <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">https://www.wikidata.org/wiki/Wikidata:Identifiers</rdfs:isDefinedBy>
+    </owl:DatatypeProperty>
```

Comment:
here is no `rdfs:domain` and no `rdfs:range`, correct?

Reply:
Yes. It was discussed in https://wiki.lyrasis.org/display/VIVO/2025-01-22+Ontology+Interest+Group+Call that rdfs:range is causing more trouble that it would bring benefits. 

Regarding rdfs:domain, Wikidata IDs can be provided for instances of basically all classes. A domain would not make sense here, I guess.